<launch>
  <!-- Arguments -->
  <arg name="cmd_vel_topic" default="p3dx/cmd_vel" />
  <arg name="odom_topic" default="p3dx/odom" />
  <arg name="move_forward_only" default="false"/>

  <!-- move_base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find p3dxbot)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find p3dxbot)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find p3dxbot)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find p3dxbot)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find p3dxbot)/param/move_base_params.yaml" command="load" />
    <remap from="cmd_vel" to="$(arg cmd_vel_topic)"/>
    <remap from="odom" to="$(arg odom_topic)"/>
    <remap from="map" to="/map"/>
  </node>
</launch>
